server/src/infra/repositories/smart-info.repository.ts

```diff
@@ -61,6 +61,7 @@ export class SmartInfoRepository implements ISmartInfoRepository {
         .andWhere('a.fileCreatedAt < NOW()')
         .leftJoinAndSelect('a.exifInfo', 'e')
         .orderBy('s.embedding <=> :embedding')
+        .orderBy('a.fileCreatedAt DESC')
```

Comment:
I don't think this is a good idea. For one thing, this code doesn't work as expected because it replaces the previous order by instead of using `addOrderBy`. But it wouldn't work even if changed. This is because vector search does comparisons on float values that are very unlikely to be equal.

It's possible to make this query a CTE and order it by date in the second pass, but this is also not great. Instead of sorting by relevance, it can now show less relevant images first. Additionally, sorting by date would be a real headache once we add pagination for this query.

Reply:
You're right.
I removed it.